7.1.цикл_for/7.1.8.py

```diff
@@ -0,0 +1,3 @@
+n=int(input())
+for i in range(n + 1):
+    print('Квадрат числа', i, 'равен', i ** 2)
```

Comment:
Все ок, но лучше использовать f-строки:
```suggestion
    print(f"Квадрат числа {i} равен {i ** 2}")
```